node/README.md

```diff
@@ -235,7 +235,59 @@ that you need a [cheatsheet](https://github.com/wsargent/docker-cheat-sheet#expo
 
 ### 5.2 Bootstrapping a Private Network
 
-TBD
+It is possible to set up a private RChain network by running a standalone node and using it for bootstrapping other nodes. Here we
+run one on port 4000:
+
+```
+$ java -Djava.net.preferIPv4Stack=true -jar /Users/rabbit/projects/rchain/node/target/scala-2.12/rnode-assembly-0.1.3.jar -s -p 4000
```

Comment:
Bit too late but this points to your user directory. I know this is obvious but some people might end up looking the `/Users/rabbit` catalog.